src/core/public/chrome/ui/_loading_indicator.scss

```
@@ -11,7 +11,7 @@ $kbnLoadingIndicatorColor2: tint($euiColorAccent, 60%);
   top: 0; // 1
   left: 0; // 1
   right: 0; // 1
-  z-index: $euiZLevel1; // 1
+  z-index: $euiZLevel1 + 10; // 1
```

Can we just bump this up a whole other level?

```suggestion
  z-index: $euiZLevel2; // 1
```

---

I love z-index escalations issues.